npm/grep/scripts.js

```diff
@@ -0,0 +1,170 @@
+const { exec, echo, exit } = require('shelljs')
+
+function runCommand (command) {
+  if (exec(command).code !== 0) {
+    echo(`Error: ${command}`)
+    exit(1)
+  }
+}
+
+// Run e2e tests 🧪
+runCommand('npx cypress-expect --min-passing 5 --pending 0')
```

Comment:
How come these aren't leverage the same system-test setup as the other `run` mode tests in the repo?

Reply:
We need to use `cypress-expect` to validate the results because we aren't just checking if tests pass or not

Reply:
@jordanpowell88 what else is being verified? The expect json seem to be verifying the pass/fail results.

Reply:
We need essentially a way to test the results of each test ie: this test passed, this test is pending, this test skipped etc

Reply:
We have examples where do we this in the system tests: https://github.com/cypress-io/cypress/blob/develop/system-tests/test/testConfigOverrides_spec.ts#L42

Reply:
@emilyrohrbough I can work on changing the structure of these, but it will be a heavy lift. stay tuned